### PR TITLE
Use EnumUtils

### PR DESCRIPTION
**Pull request by MegaPiggy**

### CODE REVIEW

**Review by JohnCorby — APPROVED** [review]

File: NewHorizons/Builder/General/SectorBuilder.cs [hunk 1]
```diff
@@ -22,7 +25,7 @@ public static Sector Make(GameObject planetBody, OWRigidbody owRigidBody, float
             sectorGO.AddComponent<OWTriggerVolume>();
 
             Sector S = sectorGO.AddComponent<Sector>();
-            S._name = (Sector.Name)24;
```
Comment (xen-42) [inline NewHorizons/Builder/General/SectorBuilder.cs:25]: I believe OWO was hardcoded to expect NH to use this value, so we'd have to coordinate this with Vesper. Maybe.

**Review by xen-42 — CHANGES_REQUESTED** [review]

**Comment by xen-42** [comment]
Tested with Signals+ and Solar Neighbourhood and no custom signals had any sound, both ones that load external sound files and those that use base game sounds. Haven't tested audio volumes / slides with sound effects but I imagine the same might apply to them.

**Comment by MegaPiggy** [comment]
TryParse fixed xen being deaf

File: NewHorizons/Builder/General/SectorBuilder.cs (hunk 1 above)
Comment (MegaPiggy) [reply NewHorizons/Builder/General/SectorBuilder.cs:25]: I've checked the code thoroughly and there is no 24 anywhere.

File: NewHorizons/Builder/General/SectorBuilder.cs [hunk 2]
```diff
@@ -22,7 +25,7 @@ public static Sector Make(GameObject planetBody, OWRigidbody owRigidBody, float
...
+            S._name = Name;
```
Comment (xen-42) [inline NewHorizons/Builder/General/SectorBuilder.cs:28]: this breaks OWO

**Review by xen-42 — CHANGES_REQUESTED** [review]

File: NewHorizons/Builder/General/SectorBuilder.cs (hunk 2 above)
Comment (MegaPiggy) [reply NewHorizons/Builder/General/SectorBuilder.cs:28]: no it doesn't. OWO doesn't use (Sector.Name)24.

**Review by JohnCorby — APPROVED** [review]

**Review by xen-42 — APPROVED** [review]

**Pull request merged**